File: Code/3DMANAGER-APP/3dmanager-app.client/src/models/group/GroupDashboardData.ts
export interface PrinterHoursObject {
    printerId: number;
    printerName: string;
    printerHours: string;
}

export interface GroupDashboardData {
    groupTotalHours: string;
    groupTotalPrints: number;
    groupTotalFilament: number;
    groupUserCount: number;
    groupFilamentCount: number;
    groupPrinterCount: number;
    groupPrinterHours: PrinterHoursObject[];
}

File: Code/3DMANAGER-APP/3DMANAGER-APP.BLL/Models/Group/GroupDashboardData.cs
namespace _3DMANAGER_APP.BLL.Models.Group
{
    public class PrinterHoursObject
    {
        public int PrinterId { get; set; }
        public string PrinterName { get; set; }
        public string PrinterHours { get; set; }
    }
    public class GroupDashboardData
    {
        public string GroupTotalHours { get; set; }
        public int GroupTotalPrints { get; set; }
        public decimal GroupTotalFilament { get; set; }
        public int GroupUserCount { get; set; }
        public int GroupFilamentCount { get; set; }
        public int GroupPrinterCount { get; set; }
        public List<PrinterHoursObject> GroupPrinterHours { get; set; }

    }
}
